Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM php:7.0-apache
+
+LABEL maintainer="Kitware, Inc. <software.kitware.com>"
+
+RUN apt-get update                                                             \
+ && apt-get install -y gnupg                                                   \
+ && curl -sL https://deb.nodesource.com/setup_6.x | bash                       \
+ && apt-get install -y git libbz2-dev libfreetype6-dev libjpeg62-turbo-dev     \
+    libmcrypt-dev libpng-dev libpq-dev libxslt-dev libxss1 nodejs unzip wget   \
+    zip                                                                        \
+ && docker-php-ext-configure pgsql --with-pgsql=/usr/local/pgsql               \
+ && docker-php-ext-configure gd --with-freetype-dir=/usr/include/              \
+                                --with-jpeg-dir=/usr/include/                  \
+ && docker-php-ext-install -j$(nproc) bcmath bz2 gd pdo_mysql pdo_pgsql xsl    \
+ && docker-php-ext-enable xdebug                                               \
```

Comment:
:straggler:

Reply:
@opadron xdebug is set as a require-dev in the composer.json file, and I didn't realize this so I apologize for overlooking it, but it means that you would also need to use the `--no-dev` flag with composer install, e.g.:
```bash
composer install --no-interaction --no-progress --prefer-dist --no-dev
```

Reply:
@opadron I'm getting the following error from `docker-compose up --build`

```bash
cdash_1  | /docker-entrypoint.sh: line 2: $'\r': command not found
': not a valid identifieroint.sh: line 3: declare: `__exit_callbacks
cdash_1  | /docker-entrypoint.sh: line 4: syntax error near unexpected token `$'{\r''
'dash_1  | /docker-entrypoint.sh: line 4: `onexit() {
mysql_1  | 180719 22:09:12 [Note] mysqld (mysqld 5.5.60) starting as process 66 ...
mysql_1  | [Entrypoint] Waiting for server...
cdash_cdash_1 exited with code 2
```